core/rpc/core/src/impl/query.rs

```diff
@@ -205,11 +216,11 @@ impl<C: CkbRpc> MercuryRpcImpl<C> {
         search_key: indexer::SearchKey,
         order: indexer::Order,
         limit: Uint64,
-        after_cursor: Option<Bytes>,
+        after_cursor: Option<u64>,
     ) -> InnerResult<indexer::PaginationResponse<indexer::Cell>> {
         let pagination = {
             let order: common::Order = order.into();
```

Comment:
we have import the `common::Order` already, suggest to remove the prefix or import the `PaginationRequest` and use one line:
```
let pagination = PaginationRequest::new(after_cursor, order.into, Some(limit.into()), None, false);
```